app/networkapi/templates/filebrowser/upload.html

```diff
@@ -19,7 +19,12 @@
 {% block javascripts %}
     {{ block.super }}
     <script type="text/javascript" src="../../jsi18n/"></script>
+
+    <!-- override: start #160 -->
+    <!-- <script type="text/javascript" src="{{ settings_var.URL_FILEBROWSER_MEDIA }}js/upload.js"></script> -->
     <script type="text/javascript" src="{% static 'js/filebrowser/upload.js' %}"></script>
```

Comment:
since this is used in lots of other places, too, better to replace those `'` quotes around the static tag with `"`, to keep the template code uniform.